### PR TITLE
Adapt the frontend lambda to the subgraph definition

### PR DESCRIPTION
**Pull request by chrisdotn**
The subgraph definition and the query from the frontend did not match. This PR adapts the frontend lambda so that the query conforms to the schema from the subgraph definition. In particular, it adapts the token part of the query and 

- kills `tokenUri` entirely
- renames `metadata` to `ipfsUri`
- kills `attributes` entirely.

### CODE REVIEW

File: module1/blockchain-handler/lib/erc721/queryHandler.ts [hunk 1]
```diff
@@ -317,36 +302,36 @@ export async function queryGraphForNFT(
...
+//   image,
+//   attributes
+// }: GraphToken['metadata']): SentenceNftMetdata {
+//   return {
+//     name,
+//     description,
+//     image,
+//     attributes: attributes.map(({ key, value }) => ({ trait_type: key, value }))
+//   };
+// }
```
Comment (the-rpc) [inline module1/blockchain-handler/lib/erc721/queryHandler.ts:317]: @chrisdotn Should we remove commented out code?

**Comment by the-rpc** [comment]
Changes look good. Approval pending a test deployment.

**Comment by chrisdotn** [comment]
end to end test successful with e2e deplyoment script

**Review by dpdornseifer — APPROVED** [review]
lgtm - would suggest @chrisdotn to remove the commented out code before merging the code

**Pull request merged**